apps/api/src/app/feeds/feeds.controller.ts

```diff
@@ -68,6 +69,7 @@ export class FeedsController {
     );
   }
 
+  // TODO: change feedId type to mongodbId
```

Comment:
I think that instead of adding all those TODOs, we should add tickets for those tasks.

Reply:
yeah sure I will create a task in linear with all those apis

Reply:
https://linear.app/novu/issue/NV-2597